src/main/java/com/google/sps/servlets/LogoutServlet.java

```diff
@@ -21,16 +21,55 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
+import javax.servlet.http.Cookie;
+import com.google.appengine.api.utils.SystemProperty;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that invalidates the user's session after they log out. */
 @WebServlet("/logout")
 public class LogoutServlet extends HttpServlet {
 
     @Override
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //log the user out if we're on the development server
+        if (SystemProperty.environment.value() != SystemProperty.Environment.Value.Production) {
```

Comment:
Does the development server here refer to the deployed or local version? Also, what does .Production do?

Reply:
SystemProperty.Environment.Value.Development is the local version and SystemProperty.Environment.Value.Production is the deployed version.